docs/quickstart/setup_quickstart.md

```diff
@@ -72,6 +72,15 @@ $ cd build-target               # this is where Flink is installed to
 ~~~
 {% endif %}
 
+### Alternatively
```

Comment:
Yes. And perhaps use the `<div class="codetabs" markdown="1">` formatting as with `dataset_transformation.md` and elsewhere. With tabs it is easy to see the different choices.